lib/ssl/src/ssl_handshake.hrl

```diff
@@ -441,4 +440,29 @@
 
 -record(signature_algorithms_cert, {signature_scheme_list}).
 
--endif. % -ifdef(ssl_handshake).
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% OCSP stapling
+%% RFC 6066 section 8 Certificate Status Request
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
```

Comment:
I think this section belongs to the RFC 6066 section further up in this file. We forgot to update to comment there, max_fragment_length is already supported.

Reply:
> I think this section belongs to the RFC 6066 section further up in this file. We forgot to update to comment there, max_fragment_length is already supported.

done